packages/in-percy/index.js

```diff
@@ -0,0 +1,6 @@
+// Returns true if executing in the Percy rendering environment
+export default function inPercy() {
```

Comment:
Optional potential simplification:
```javascript
let hostname = typeof window == 'object' && window.location && window.location.hostname;
return (hostname || '').match('proxyme.percy.io');
```

Separately and not very important right now, I think single-line `if` statements are against our JS style in other places (correctly so IMO). It'd be nice to get the linter applied in this repo at some point to start pushing stuff that direction, though understand it's a large refactor.